src/http/api/account.ts

```diff
@@ -121,30 +126,51 @@ export function createGetAccountFollowsHandler(
     return async function handleGetAccountFollows(ctx: AppContext) {
         const site = ctx.get('site');
 
-        // Validate input
         const handle = ctx.req.param('handle') || '';
-
         if (handle === '') {
             return new Response(null, { status: 400 });
         }
 
         const type = ctx.req.param('type');
-
         if (!['following', 'followers'].includes(type)) {
             return new Response(null, { status: 400 });
         }
 
         const siteDefaultAccount = await accountRepository.getBySite(site);
 
-        // Get follows accounts and paginate
-        const queryNext = ctx.req.query('next') || '0';
-        const offset = Number.parseInt(queryNext);
+        const queryNext = ctx.req.query('next');
+        const next = queryNext ? decodeURIComponent(queryNext) : null;
 
-        const accountFollows = await accountFollowsView.getFollows(
-            type,
-            siteDefaultAccount,
-            offset,
-        );
+        let accountFollows: AccountFollows;
+
+        if (handle === 'me') {
+            accountFollows = await accountFollowsView.getFollowsByAccount(
+                siteDefaultAccount,
+                type,
+                Number.parseInt(next || '0'),
+                siteDefaultAccount,
+            );
+        } else {
+            const ctx = fedifyContextFactory.getFedifyContext();
+            const apId = await lookupAPIdByHandle(ctx, handle);
```

Comment:
I didn't move this lookup to the view because I want the account 
`const account = await accountRepository.getByApId(new URL(apId));` and this account comes from `accountRepository`. Didn't want to duplicate the whole logic of `accountRepository.getByApId` in the view.